data/blog/app-feedback-program.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Electron App Feedback Program
+author: sofianguy
+date: '2018-10-04'
+---
+
+Electron is working on building a streamlined release process and having faster releases. To help with that, we started the App Feedback Program for large-scale Electron apps to test our beta releases and report app-specific issues to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible.
+
+## Who can join?
+Our criteria and expectations for apps joining this program include the following items:
+- Test your app during the beta period for 10,000+ user-hours.
+- Have a single point-person to speak for the app's Electron bugs, app blockers, and check-in weekly, either during our regularly scheduled online meeting, or asynchronously with @sofianguy
```

Comment:
`@sofianguy` could be made into a link for convenience.

Reply:
Odd transition from a list of nouns to a verb. Maybe:

-  Have a single point-person who will check in weekly to discuss the app's Electron bugs and app blockers

Reply:
Also, on @zeke's CoC comment below, this is probably where the CoC should be mentioned

Reply:
I would take your name/handle out of the post, just tell folks they'll need to asynchronously update our side.